### PR TITLE
Firewall - Add reference for raw URI fields

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses https://jira.cfops.it/browse/PCX-1868.

### CODE REVIEW

File: products/firewall/src/content/cf-firewall-language/fields.md [hunk 1]
```diff
@@ -239,6 +239,54 @@ The Cloudflare Firewall Rules language supports these standard fields:
...
       </td>
   </tr>
+  <tr id="field-raw-http-request-full-uri">
+    <td valign="top"><code>raw.http.request.full_uri</code><br /><Type>String</Type></td>
+    <td>
+      <p>Similar to the <a href="#field-http-request-full-uri"><code>http.request.full_uri</code></a> non-raw field. Represents the full URI as received by the web server without the URI fragment (if any) and without any transformation.
+      </p>
+      <p>Notes:</p>
+      <p><ul>
+      <li>This raw field includes some basic normalization done by NGINX. However, this may change in the future.</li>
```
Comment (orium) [inline products/firewall/src/content/cf-firewall-language/fields.md]: Maybe we can just say "might include some basic normalization done by our http server".  This way we don't imply that strictly follow whatever nginx does.

Also applies to the other fields documented in this PR.

File: products/firewall/src/content/cf-firewall-language/fields.md [hunk 2]
```diff
@@ -239,6 +239,54 @@ The Cloudflare Firewall Rules language supports these standard fields:
...
+      <li>Only available in expressions of <a href="https://developers.cloudflare.com/rules/transform">Tranform Rules</a>.</li>
```
Comment (orium) [inline products/firewall/src/content/cf-firewall-language/fields.md]: This is available in all phases.  I think we can just remove this line entirely.

Also applies to the other fields documented in this PR.

File: products/firewall/src/content/cf-firewall-language/fields.md [hunk 3]
```diff
@@ -239,6 +239,54 @@ The Cloudflare Firewall Rules language supports these standard fields:
...
+      </ul></p>
+  </td>
+  </tr>
+  <tr id="field-raw-http-request-uri">
+    <td valign="top"><code>raw.http.request.uri</code><br /><Type>String</Type></td>
+    <td>
+      <p>Similar to the <a href="#field-http-request-uri"><code>http.request.uri</code></a> non-raw field. Represents the absolute URI of the request without any transformation.
```
Comment (orium) [inline products/firewall/src/content/cf-firewall-language/fields.md]: > Represents the absolute URI of the request without any transformation.

This is just the path and query.

File: products/firewall/src/content/cf-firewall-language/fields.md (hunk 2 above)
Comment (pedrosousa) [reply products/firewall/src/content/cf-firewall-language/fields.md]: Thanks. I had understood differently from the content in our internal wiki.

File: products/firewall/src/content/cf-firewall-language/fields.md (hunk 3 above)
Comment (pedrosousa) [reply products/firewall/src/content/cf-firewall-language/fields.md]: Thanks. Also updated the description of the `http.request.uri` field.

**Review by orium — APPROVED** [review]

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**